res/drawable/profile_picture_view_small_background.xml

```diff
@@ -6,6 +6,4 @@
     <solid  android:color="@color/unimportant" />
 
     <corners android:radius="18dp" />
-
-    <stroke android:width="@dimen/profile_picture_border_thickness" android:color="@color/border" />
```

Comment:
The placeholders in the design actually have borders

Reply:
Yep, I took the liberty to remove it to show the difference. Most of the modern apps (messengers at least) that use circular avatars don't have borders and it looks cleaner IMHO. Please let me know if think we should stick to the old design, no problem going back.